test-broken-links.js

```diff
@@ -0,0 +1,72 @@
+const fs = require("fs");
```

Comment:
let's rename this file to `check-legend-application-documentation-broken-links.js`, also move it to `scripts/check-legend-application-documentation-broken-links.js` to start organize and not pollute the top-level of the repo.